e2e/app/page/admin-user-profile-info.ts

```diff
@@ -144,7 +144,7 @@ export default class UserProfileInfo extends AuthenticatedPage {
     const regexpSize = /\$([0-9.]+) used of \$([0-9.]+) limit/;
     const match = initialCreditsUsedValue.match(regexpSize);
     // extract the credit limit value
-    const creditLimitValue = "$" + `${match[2]}`;
+    const creditLimitValue = '$' + `${match[2]}`;
```

Comment:
'$' + `${match[2]}` had to add back the $ sign to match it with the initial-credit-limit value in [line 174](https://github.com/all-of-us/workbench/pull/6139/commits/c7e20705a5933bb0c1af474259751d59dce0a6ee#diff-589375b4fc33f41fa9897ce10f70e5c59b25225af4ec04c1cc93b0f7d8f9734bR174).